src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -220,6 +220,24 @@
       <arg type="s" name="transaction_address" direction="out"/>
     </method>
 
+   <!-- Available options:
+        "reboot" (type 'b')
+    -->
+    <method name="KernelArgs">
+      <arg type="s" name="existing_kernel_arg_string"/>
```

Comment:
Is this value actually used by the daemon?  It looks to me like we pass it all the way down but don't actually use it, rather it's just parsed on the client side.   Is that correct?  If so, let's just drop this from the DBus API?

Reply:
Hey, thanks for the review :). We actually do use it, it is this line
``` 
__attribute__((cleanup(_ostree_kernel_args_cleanup))) OstreeKernelArgs *kargs = _ostree_kernel_args_from_string (self->existing_kernel_args);
```

We use this attribute so that we do not need to do the extra loading of the wanted kargs again in the transaction execute, as we have already done the loading in `rpmostree_os_call_get_deployment_boot_config_sync` function.

Reply:
Ah, right OK.